puppet/modules/site_couchdb/manifests/init.pp

```diff
@@ -41,8 +41,8 @@
   $couchdb_pwhash_alg       = $couchdb_config['pwhash_alg']
 
   if $couchdb_mode == 'multimaster' { include site_couchdb::bigcouch }
-  if $couchdb_mode == 'master'      { include site_couchdb::master }
-  if $couchdb_mode == 'mirror'      { include site_couchdb::mirror }
+  if $couchdb_mode == 'plain'       { include site_couchdb::plain }
+  # if $couchdb_mode == 'mirror'      { include site_couchdb::mirror }
```

Comment:
hugh, won't this break current provider configs that have `couch.mode='master'` ?